Calendar/Get-CalendarDiagnosticObjectsSummary.ps1

```diff
@@ -257,6 +256,46 @@ function GetMailbox {
     }
 }
 
+<#
+.SYNOPSIS
+Checks if a set of Calendar Logs is from the Organizer.
+#>
+function SetIsOrganizer {
+    param(
+        $CalLogs
+    )
+    $IsOrganizer = $false
+
+    foreach ($CalLog in $CalLogs) {
+        if ($CalendarItemTypes.($CalLog.ItemClass) -eq "IpmAppointment" -and
+            $CalLog.ExternalSharingMasterId -eq "NotFound" -and
+            ($CalLog.ResponseType -eq "1" -or $CalLogs.ResponseType -eq "Organizer")) {
+            $IsOrganizer = "True"
+            Write-Verbose "IsOrganizer: [$IsOrganizer]"
+            return $IsOrganizer
+        }
+    }
+    Write-Verbose "IsOrganizer: [$IsOrganizer]"
+    return $IsOrganizer
+}
+
+function SetIsRoom {
```

Comment:
function is created, but never used.

Reply:
Both of these functions are for a new feature that Rui is adding, direct export to Excel using OpenXML with formatting based of if they are the Organizer or if this is a Room MB supplied with these functions.

Reply:
Okay, if it isn't going to be used till a later date with some other changes, it should be checked in at that time so everything can be reviewed all at once and have a better understanding of what is occurring.

Reply:
Now is on Line 1612.